### ISSUE TITLE
Can't break list inside into 2 parts

### ISSUE THREAD

**Issue by Gecko81** [froala/wysiwyg-editor#3930]
##### Expected behavior.

Get a line without bullet point after pasting empty lines by "Enter" inside list.

##### Actual behavior.

Creates multiple empty list lines with bulletpoint and there is no way to break into 2 parts.

##### Steps to reproduce the problem.

https://jsfiddle.net/Gecko81/c7u5t014/3/

0. Create unordered list with few lines
0. Place caret at the end of the first line
0. Hit enter multiple times
0. Hit backspace

##### Editor version.

2.9.8

##### OS.

Ubuntu 19.10 eoan

##### Browser.

Mozilla Firefox 76.0.1 (64-bit)

##### Recording.

![2020-05-22_15-27](https://user-images.githubusercontent.com/23085953/82668227-fb3b5b80-9c41-11ea-8ffc-2ab02140750e.png)

![froala_list_220520](https://user-images.githubusercontent.com/23085953/82668235-0098a600-9c42-11ea-9c05-55cd9a0c2f08.gif)

**Comment by deanmikan** [comment]
Agreed, this is a very prominent issue for users. Would love to see this fixed. 
The backspace key should not go back to the previous list item but remove the list item and outdent like it almost all regular document editors.

**Comment by ilyaskarim** [comment]
We’re closing this issue due to inactivity. If you’re still experiencing this problem on the latest version of our editor, please feel free to reopen the issue and let us know. You can find details on the latest updates here: [Froala Editor Changelog](https://froala.com/wysiwyg-editor/changelog/).
